examples/regexp-match-operations/regexp_match_operations.md

```diff
@@ -0,0 +1,12 @@
+# RegExp operations
+
+The `RegExp` type include a set of langlib functions for matching patterns in strings, enabling operations to find, validate, group, and extract data based on regular expressions.
```

Comment:
```suggestion
The `RegExp` type support a set of langlib functions to match patterns in strings and enable operations such as finding, validating, grouping, and extracting data based on regular expressions.
```

Double check grouping?

Reply:
Also lang library vs langlib doesn't seem to be consistent across BBEs?

Reply:
Grouping is supported by match `fullMatchGroups` `matchGroupsAt` operations right?